server/utils/jwtToken.js

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I think this function could be converted into a middleware that adds the user Id to the request object and others can obtain it more easily.  Something like this:
```
const getUserId = (req, res, next)=>{
    req.userId = jwt(req.cookies.app_auth_token);
    next()
}
```